src/Land/Model/SoilWaterParameterizations.jl

```diff
@@ -421,29 +424,44 @@ pressure head ψ; meters.
 """
 hydraulic_head(z, ψ) = z + ψ
 
+"""
+    volumetric_liquid_fraction(
+        ϑ_l::FT,
+        porosity::FT,
+    ) where {FT}
+
+Compute the volumetric liquid fraction from the porosity and the augmented liquid
+fraction.
+"""
+function volumetric_liquid_fraction(ϑ_l::FT, porosity::FT) where {FT}
+    if ϑ_l < porosity
+        θ_l = ϑ_l
+    else
+        θ_l = porosity
+    end
+    return θ_l
+end
 
 
 """
     effective_saturation(
         porosity::FT,
-        θ_l::FT
+        ϑ_l::FT
     ) where {FT}
 
 Compute the effective saturation of soil.
 
-
-`θ_l` is defined to be zero or positive. If `θ_l` is negative, 
+`ϑ_l` is defined to be zero or positive. If `ϑ_l` is negative, 
 hydraulic functions that take it as an argument will return 
 imaginary numbers, resulting in domain errors. Exit in this 
 case with an error.
-
 """
-function effective_saturation(porosity::FT, θ_l::FT) where {FT}
+function effective_saturation(porosity::FT, ϑ_l::FT) where {FT}
 
-    if θ_l < 0
-        throw(DomainError(θ_l, "Effective saturation is negative."))
+    if ϑ_l < 0
+        throw(DomainError(ϑ_l, "Effective saturation is negative."))
```

Comment:
This may not be GPU friendly. I'd just use `error` (see [this](https://github.com/CliMA/ClimateMachine.jl/blob/ck/update_heat_eq/src/Common/Thermodynamics/relations.jl#L1107-L1129) for GPU-friendly error handling).

Reply:
will take on next!